inst/workflow/data_mapping.yaml

```diff
@@ -1,141 +1,54 @@
 meta:
   File: mapping.yaml
   description: Data Mappings to prepare for metric calculations
-spec:
```

Comment:
So, we are no longer putting any specs/requirements on the raw data? This spec section seems like it could be a useful doc for specifying and updating data requirements from Raw+. Now, all raw data requirements are inside the queries below. I may have missed a discussion, and I don't have strong preferences on this, but it seems a bit more obfuscated for the dev systems team to access the requirements in this way.

Reply:
Totally hear what you're saying - not sure where the full data spec should live, if anywhere.  I like the idea of keeping the metric workflows self-contained, and also avoiding maintaining a spec in two (or more) places.  There's a step missing between the metric workflows and mapping workflow that grabs and collates metric-level specs into a single mapping data spec (also possible y'all have already addressed this).

Reply:
I think the `CombineSpecs()` utility function handles the last part of your comment, if I understand correctly. 

re: a full data spec (including raw data) i think you are right that it shouldn't live in the metrics and that the metrics should have all of the spec information that they need so that they are transportable (assuming the definition of `Mapped_*` data doesn't change).